src/integration-tests/tests/driver.rs

```diff
@@ -47,6 +47,18 @@ mod driver {
             .map_err(integration_tests::report_error)
     }
 
+    #[test_case(pubsub::client::TopicAdmin::builder(); "default")]
+    #[test_case(pubsub::client::TopicAdmin::builder().with_tracing(); "with tracing enabled")]
+    #[test_case(pubsub::client::TopicAdmin::builder().with_retry_policy(retry_policy()); "with retry enabled")]
```

Comment:
Consider simplifying this.  We have been cargo-culting these since I created the original integration tests for secret manager, it made sense at the time to make sure all the generated code paths were tested, but by now?  Probably one test with the retry policy enabled is good enough.

Reply:
Simplified to just use the default.